app/models/dialog_field_sorted_item.rb

```diff
@@ -91,7 +91,7 @@ def sort_data(data_to_sort)
   end
 
   def raw_values
-    @raw_values ||= dynamic ? values_from_automate : self[:values].to_miq_a
+    @raw_values ||= dynamic ? values_from_automate : static_raw_values
     unless @raw_values.collect { |value_pair| value_pair[0] }.include?(default_value)
```

Comment:
@eclarizio - I would second @gmcculloug - Can we extract this out into a private method as well - I'm getting lost in what the block is trying to accomplish.